src/background/notification.background.ts

```diff
@@ -192,6 +192,10 @@ export default class NotificationBackground {
         }
 
         if (await this.vaultTimeoutService.isLocked()) {
+            if (!(await this.allowPersonalOwnership())) {
```

Comment:
The enclosing brackets are unnecessary:

```suggestion
            if (!await this.allowPersonalOwnership()) {
```

You could also fix up the check below while you're here (line 213).